### PR TITLE
Update Docker image and README instructions

### PR DESCRIPTION
**Pull request by marcofavorito**
## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,40 +43,52 @@ RUN HOME=/home/default && \
...
+    rm -rf Syft
 
-#RUN wget http://ftp.us.debian.org/debian/pool/main/b/bison/libbison-dev_3.0.4.dfsg-1+b1_amd64.deb &&\
-#  wget http://ftp.us.debian.org/debian/pool/main/b/bison/bison_3.0.4.dfsg-1+b1_amd64.deb &&\
-#  sudo dpkg -i libbison-dev_3.0.4.dfsg-1+b1_amd64.deb &&\
-#  sudo dpkg -i bison_3.0.4.dfsg-1+b1_amd64.deb
+USER default
 
+#RUN git clone --recursive https://github.com/whitemech/lydia.git /home/default/lydia
+COPY --chown=default:default . /build/lydia
```
[the remark this answers is not in the document]
Comment (francescofuggitti) [reply Dockerfile]: can we use clone now?

[the remark this answers is not in the document]
Comment (marcofavorito) [reply Dockerfile]: yes it is public. But we need to checkout to `master`

File: README.md [hunk 2]
```diff
@@ -5,33 +5,102 @@
...
+  -s                          Print the summary.
+[Option Group: formula]
+   
+  [At least 1 of the following options are required]
+  Options:
+    --ldlf TEXT:FILE Excludes: --ltlf
+                                File to an LDLf formula.
+    --ltlf TEXT:FILE Excludes: --ldlf
+                                File to an LTLf formula.
+```
```
[the remark this answers is not in the document]
Comment (francescofuggitti) [reply README.md:40]: just to remember, this has to be changed accordingly later

**Review by francescofuggitti — APPROVED** [review]
LGTM

**Pull request merged**